pkg/features/list.go

```diff
@@ -94,6 +94,9 @@ var (
 	// ScannerV4 enables various capabilities associated with ACS + Clair consolidated scanner.
 	ScannerV4 = registerUnchangeableFeature("Enable Scanner V4 Support", "ROX_SCANNER_V4", true)
 
+	// ScannerV4Enabled indicates Scanner V4 is installed and should be used as the default image scanner in Central/Sensor.
+	ScannerV4Enabled = registerFeature("Enables Scanner V4 runtime functionality", "ROX_SCANNER_V4_ENABLED", false)
```

Comment:
This is no longer an env var becuase there is no opt-in/opt-out, right?

Reply:
With the new `registerFeature` compared to the `registerUnchangeableFeature` you can now also opt-in / opt-out as well within release builds (it essentially behaves like an env var).

Reply:
> This is no longer an env var because there is no opt-in/opt-out, right?

This is a changeable feature flag so technically it is opt-in/opt-out - the main reason for moving it to a feature was so that the UI can query `/v1/featureflags` and react accordingly (ie: hide the V4 tile / vuln status / cert status / etc. if needed). 

There is no opt-in/opt-out at runtime and Scanner V4 will be installed by default - but the installers will still allow installing without Scanner V4, and in case of an emergency setting the enabled feature to `false` should disable all associated functionality (this is explained in more detail in the design)
--
As a separate note, I am proposing of changing the feature `ScannerV4` to `ScannerV4Support`, and then changing `ScannerV4Enabled` to `ScannerV4` - believe that will read better in code and make more sense long term.